.github/workflows/release.yaml

```diff
@@ -21,6 +21,6 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.0.0
+        run: ./hack/cr-wrapper.sh
```

Comment:
Could you add a comment here from the section `What this PR does`?

Reply:
Done

Reply:
@brtknr it would be good to add , issue id or PR # , so that we can check later and add as TODO here: revery once ... PR or Issue is fixed.

Reply:
Done